tests/data/Block.2.sample1.json

```diff
@@ -1,71 +1,74 @@
 {
-    "header": {
-        "prev_block": "0x2ba067da82aa7a02c49771cc07146d786b3840585c6125d79e4b4914c62347a4bc900c8b83dc369d0d3a45493549086926ada889d532ccde4b9ee2e3d12d4844",
-        "height": "2",
-        "merkle_root": "0x5ed0db3335189a96f23b00787bc4e34d7f0c20ab9536f3e951c9049299e5f2509656f858fc0a6ed1851948935f7a19bbce9170e3ae767129aa017bdd3001dfc5",
-        "validators": "[]",
-        "signature": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-        "enrollments": [],
-        "random_seed": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-        "missing_validators": []
+    "header":{
```

Comment:
Please print out JSON Pretty and make it easier to compare with previous data.
https://jsonformatter.org/json-pretty-print